mscore/palette.cpp

```diff
@@ -792,28 +797,31 @@ void PaletteScrollArea::keyPressEvent(QKeyEvent* event)
       QScrollArea::keyPressEvent(event);
       }
 
-//---------------------------------------------------------
-//   mouseDoubleClickEvent
-//---------------------------------------------------------
-
-void Palette::mouseDoubleClickEvent(QMouseEvent* ev)
+void Palette::mouseReleaseEvent(QMouseEvent *event)
       {
-      if (_disableDoubleClick)
+      pressedIndex = -1;
+
+      update();
+
+      if (_disableElementsApply)
             return;
-      int i = idx(ev->pos());
-      if (i == -1)
+
+      int index = idx(event->pos());
+
```

Comment:
You don't need to put so many empty lines in a function... Like I would not put one here. Too many empty lines make the code unclear.